.travis.yml

```diff
@@ -104,8 +102,6 @@ jobs:
     - name: "Deploy Price Estimator"
       if: (type != pull_request) AND (tag is present OR branch = master)
       rust: stable
-      before_script:
```

Comment:
How come ci/setup_contracts.sh is not needed here?

Reply:
`scripts/setup_contracts.sh` were not needed either. I just noticed when I was changing it to use the new script.